next-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference types="next" />
```

Comment:
This file should not be in this PR; we've since added it to `.gitignore`.  Delete it and merge master

Reply:
I've done this but it still shows in Github PR for some reason